app/controllers/administrate/application_controller.rb

```diff
@@ -5,6 +5,7 @@ class ApplicationController < ActionController::Base
     def index
       search_term = params[:search].to_s.strip
       resources = Administrate::Search.new(resource_resolver, search_term).run
+      resources = resources.includes(*resource_includes) if resource_includes.any?
```

Comment:
Well, I thought about moving it to the collection class, but that didn't work out the way I thought it would. Instead, it's been moved to the dashboard and we pass the includes from there.